tests/plugins/twilio_test.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+import pytest
+
+from detect_secrets.plugins.twilio import TwilioKeyDetector
+
+
+class TestTwilioKeyDetector(object):
+
+    @pytest.mark.parametrize(
+        'payload, should_flag',
+        [
+            (
+                'SKxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+                True,
+            ),
+            (
+                'ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
+                True,
+            ),
+        ],
+    )
+    def test_analyze(self, payload, should_flag):
+        logic = TwilioKeyDetector()
+        output = logic.analyze_line(payload, 1, 'mock_filename')
+        assert len(output) == int(should_flag)
```

Comment:
Probably can just do:

```
assert output
```

as that will make sure that it is a non-empty dictionary.